examples/07-Train-traditional-ML-models-using-the-Merlin-Models-API.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>Revise typo as: "There are two ways we can train an implicit model."</p><p>Revise typo as: "There are two implicit models you can train with Merlin Models:"</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/911/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>corrected</p><p> </p>

Reply:
<p>There is one more typo to fix with the word "Merli", right before the 2 bullets.</p>